### ISSUE TITLE
Recruiters - As a workshop organizer i want to appropriately use recruiters

### ISSUE THREAD

**Issue by dpcybuck** [mzbat/mirr#4]
The MIRRW needs guidelines for recruiters.  I had 2 good ones.  @YesItsKathleen  mentored me through the planning and did one interview/resume review.  A recruiter with infosec as their main job, and even better as their background gives better resume and interview guidance.  
We should build a guideline for work shop organizers to appropriately use recruiters.

**Comment by YesItsKathleen** [comment]
I agree and disagree. There have been many folks who have had infosec experience, who have not given good advice on interviews and resumes, in my experience. Resume review is tricky as it should include how to get in the front door of an organization and through the HR/Recruiting process. A guideline is good, but it is also good to include folks that are just straight HR or recruiters. This helps educate everyone.

**Comment by dpcybuck** [comment]
Yes, we have talked about that before, i listened and agree.  I edited my comment on the issue to reflect that.  
"A PERSON with infosec as their main job, and even better as their background gives better resume and interview guidance." to
A RECRUITER with infosec as their main job, and even better as their background gives better resume and interview guidance.
